docs/website/docs/index.md

```diff
@@ -25,11 +25,11 @@ considerations of mobile and edge deployments.
 
 IREE supports importing from a variety of ML frameworks:
 
-- [x] JAX
 - [x] PyTorch
```

Comment:
I assume this should be alphabetical?

Reply:
I could keep it alphabetical, sure. I'd prefer for the first link to not be to a mostly empty page though: https://iree.dev/guides/ml-frameworks/jax/.